signer/migrations/0002_create_user.up.sql

```diff
@@ -0,0 +1,18 @@
+-- Create the user if it doesn't exist
+DO
+$do$
+BEGIN
+   IF NOT EXISTS (
+      SELECT FROM pg_catalog.pg_roles
+      WHERE rolname = 'sbtc_user'
+   ) THEN
+      CREATE USER sbtc_user WITH PASSWORD 'your_password';
+   END IF;
+END
```

Comment:
Good idea. I can name these something better as well.

Reply:
Also, these are just placeholders for now. We should figure out a way to parameterize these long term, so that the individual signers can control and set their own credentials.

Reply:
Updated the README. Also added the `DATABASE_URL` env variable there.